buildSrc/src/main/kotlin/profiler.java-library.gradle.kts

```
@@ -0,0 +1,24 @@
+plugins {
+    id("java-library")
+}
+
+repositories {
+    jcenter()
+    maven {
+        url = uri("https://repo.gradle.org/gradle/repo")
+    }
+}
+
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(8))
+    }
+}
+
+project.extensions.create("versions", Versions::class.java)
```

:o: Could be simplified to
```suggestion
project.extensions.create<Versions>("versions")
```